browser/brave_profile_prefs.cc

```diff
@@ -34,10 +34,6 @@
 #include "brave/components/brave_webtorrent/browser/webtorrent_util.h"
 #endif
 
-#if !BUILDFLAG(USE_GCM_FROM_PLATFORM)
```

Comment:
cc: @jumde on this overall commit (if this hash gets lost in rebase, title of commit is `GCMChannelStatusSyncer was removed.`)

Reply:
Push notifications work, all good :)